test/jdk/java/net/httpclient/http2/FixedThreadPoolTest.java

```diff
@@ -174,7 +177,7 @@ static void streamTest(boolean secure) throws Exception {
                     return resp.body();
                 });
         response.join();
-        compareFiles(src, dest);
+        assert Files.mismatch(src, dest) < 0;
```

Comment:
Missed call to `assertFileContentsEqual`?

Reply:
Doh! :facepalm: Fixed in 5e9b30f431676c714cf81b85846f16c58c0dcdf6.